### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by tylerliu**

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -75,3 +75,101 @@ jobs:
...
+        # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+        # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DHAVE_TESTS=True -DBUILD_EXAMPLES=True
+
+      - name: Build
+        # Build your program with the given configuration
+        run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
+
+      - name: Start NFD
+        run: sudo systemctl start nfd.service
```
Comment (Pesa) [inline .github/workflows/cmake.yml]: macOS doesn't use systemd (and nfd is already started with `nfd-start` in an earlier step)

File: .github/workflows/cmake.yml [hunk 2]
```diff
@@ -75,3 +75,101 @@ jobs:
...
+        os: [ macos-10.15 ]
+        include:
+          - xcode: '12.5'
+            os: macos-11
+          - xcode: '13.2'
+            os: macos-11
+    runs-on: ${{ matrix.os }}
+    env:
+      NODE_LABELS: OSX
+      WAF_JOBS: 3
```
Comment (Pesa) [inline .github/workflows/cmake.yml]: this is unused, you can delete the whole section (in other projects, these env vars are used by the jenkins scripts)

File: .github/workflows/cmake.yml (hunk 1 above)
Comment (tylerliu) [reply .github/workflows/cmake.yml]: OK Thanks.

File: .github/workflows/cmake.yml [hunk 3]
```diff
@@ -75,3 +74,99 @@ jobs:
...
+          LDFLAGS: "-L/usr/local/opt/openssl@1.1/lib"
+
+      - name: Configure ndn-cxx
+        run: |
+          cd ..
+          git clone https://github.com/named-data/ndn-cxx
+          cd ndn-cxx
+          git checkout ndn-cxx-0.8.0
+          ./waf --color=yes configure --disable-static --enable-shared --without-osx-keychain
+          ./waf --color=yes build -j3
```
Comment (Pesa) [inline .github/workflows/cmake.yml]: the `-j` flag should be unnecessary, waf already defaults to the number of available CPU cores if I remember correctly.

File: .github/workflows/cmake.yml [hunk 4]
```diff
@@ -75,3 +74,99 @@ jobs:
...
+          cd ../NAC-ABE
+      - name: Configure NFD
+        run: |
+          cd ..
+          git clone https://github.com/named-data/NFD
+          cd NFD
+          git checkout NFD-22.02
+          git submodule update --init
+          ./waf --color=yes configure
+          ./waf --color=yes build -j3
```
Comment (Pesa) [inline .github/workflows/cmake.yml]: ditto

File: .github/workflows/cmake.yml (hunk 3 above)
Comment (tylerliu) [reply .github/workflows/cmake.yml]: fixed. Thanks.

File: .github/workflows/cmake.yml (hunk 4 above)
Comment (tylerliu) [reply .github/workflows/cmake.yml]: ditto

**Pull request merged**